keyboards/sets3n/zx60/readme.md

```diff
@@ -0,0 +1,19 @@
+# zx60
+
+![zx60](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [sets3n](https://github.com/sets3n)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```

Comment:
This information should be filled in. And information for how to enter the bootloader and flash the board should be added.